cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -817,7 +852,10 @@ impl MachInst for Inst {
         match self {
             Self::Mov_R_R { is_64, src, dst } if *is_64 => Some((*dst, *src)),
             Self::XMM_R_R { op, src, dst }
-                if *op == SseOpcode::Movss || *op == SseOpcode::Movsd =>
+                if *op == SseOpcode::Movss
+                    || *op == SseOpcode::Movsd
+                    || *op == SseOpcode::Movaps
+                    || *op == SseOpcode::Movd =>
```

Comment:
Unfortunately, we can't put `movd` here because it goes from one register class to another, and this may largely confuse register allocation. Can you remove it, please?